### PR TITLE
[android] Add llama2 example

### PR DESCRIPTION
**Pull request by niley7464**
This patch adds android application example to use llama2 model.
To use this application, nnstreamer-android's service application must be installed to the target device.

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: android/example_app/llama2/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+    <queries>
+        <package android:name="ai.nnstreamer.ml.inference.offloading" />
```
Comment (niley7464) [inline android/example_app/llama2/app/src/main/AndroidManifest.xml:5]: This part can make android application to call another application's service
https://developer.android.com/training/package-visibility/declaring

File: android/example_app/llama2/app/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    }
+    packaging {
+        resources {
+            excludes += "/META-INF/{AL2.0,LGPL2.1}"
+        }
+    }
+}
+
+dependencies {
+
```
Comment (wooksong) [inline android/example_app/llama2/app/build.gradle.kts:53]:
```suggestion
```

File: android/example_app/llama2/gradle/libs.versions.toml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+[versions]
+agp = "8.4.0-alpha06"
```
Comment (wooksong) [inline android/example_app/llama2/gradle/libs.versions.toml:2]: Any reason to use alpha?

File: android/example_app/llama2/app/src/main/java/com/example/llama2/MainActivity.kt [hunk 4]
```diff
@@ -0,0 +1,137 @@
+package com.example.llama2
```
Comment (wooksong) [inline android/example_app/llama2/app/src/main/java/com/example/llama2/MainActivity.kt:1]: Most other examples have the same package name, ```package org.freedesktop.gstreamer.nnstreamer;```. 
Q1. Is it OK with the same package names in the same repository? (I don't know)
Q2. There is nothing wrong with ```com.example.llama2``` but how about using our domain ```ai.nnstreamer.example.llama2``` ?

Comment (wooksong) [reply android/example_app/llama2/app/src/main/java/com/example/llama2/MainActivity.kt:1]: Or ```nnstreamer.example`.*``?

File: android/example_app/llama2/app/build.gradle.kts (hunk 2 above)
Comment (niley7464) [reply android/example_app/llama2/app/build.gradle.kts:53]: Removed empty line.
Thank you :)

+)  RPC failed error occurs now, so I will push it later

File: android/example_app/llama2/gradle/libs.versions.toml (hunk 3 above)
Comment (niley7464) [reply android/example_app/llama2/gradle/libs.versions.toml:2]: I just used auto generated template,
Chaged the version to "8.3.1" like NNS-Android.

File: android/example_app/llama2/app/build.gradle.kts (hunk 2 above)
Comment (niley7464) [reply android/example_app/llama2/app/build.gradle.kts]: Updated it!

File: android/example_app/llama2/app/src/main/java/com/example/llama2/MainActivity.kt (hunk 4 above)
Comment (niley7464) [reply android/example_app/llama2/app/src/main/java/com/example/llama2/MainActivity.kt:1]: A1. `example_app`'s `settings.gradle` does not have llama2 in `include` directory, so I don't think there will be a problem
But it looks something weired. How about make another example_app directory for NNS-android? 
android/example_app, android/kotlin_app (or something else)

A2. I will update the package name after separate the repository 😊

**Pull request merged**